src/types/widget.ts

```diff
@@ -431,8 +437,8 @@ export type Widget<
     | 'ais.sortBy'
     | 'ais.stats'
     | 'ais.toggleRefinement'
-    | 'ais.voiceSearch';
-
+    | 'ais.voiceSearch'
+    | 'ais.answers';
```

Comment:
the list is alphabetical

Reply:
https://github.com/algolia/instantsearch.js/pull/4581/commits/a9d46b18431465c35effd68a1ad1c965ed7c89a1